pkgs/applications/virtualization/gvisor/default.nix

```diff
@@ -76,7 +76,7 @@ in buildBazelPackage rec {
       rm -f "$bazelOut"/java.log "$bazelOut"/java.log.*
     '';
 
-    sha256 = "0myffqywbvqhax995z55jymwnwyxmp13r27kpbc7wb5fk9s4skvr";
+    sha256 = "033lnznqbw8a5y765prb60k5r7sicjvg63a2h24r2z6qkr2jn70b";
```

Comment:
Is there a way we could avoid this in the future?

Reply:
I guess that would be nice, but I don't know how to achieve it..

Reply:
buildBazelPackage uses a fixed-output-derivation, which is a crude hack around specifying all WORKSPACE dependencies and their hashes. The solution would be a `bazel2nix` tool that can read WORKSPACE files and generate a nix/json file with all of the required ones. Afaik there was some work in bazel a while ago to dump dependencies, but it didn’t go anywhere.

Reply:
You can see that we do the “correct” approach in a semi-manual fashion for bazel itself, see `src-deps.json` and `update-srcDeps.py`.

Reply:
I have a thing bazel on `bazel query` https://github.com/NixOS/nixpkgs/pull/106984/files#diff-893205af0402e7e262a4b5a6ef1fcdf7ef24bcd798171c72a514fc6f2509b7bcR322
The only issue is it needs bazel, but that's only to run updater script, so no recursion.